packages/api/lib/rulesHelpers.js

```diff
@@ -682,11 +677,28 @@ async function createRuleTrigger(ruleItem) {
   return newRuleItem;
 }
 
+async function deleteOldEventSourceMappings(knex, item) {
```

Comment:
Can we get a docstring for this?

Reply:
Can we get a docstring for this and/or add TS check to `rules.js` / this file?

Reply:
Also - it looks like the units that tested this in the model were omitted from this changeset (e.g. `test.serial('deleteOldEventSourceMappings() removes kinesis source mappings', async (t) => {` ) from https://github.com/nasa/cumulus/pull/2733    Was that deliberate? 

Given this is exported as a helper we should consider having units covering it, whatever the endpoint tests are doing.

Reply:
Yeah, that makes sense to do here. Bringing in `deleteOldEventSourceMappings` was done to address some inconsistencies with main but it was done after the model tests were ported to the helpers 🤦. 

I'll bring the tests over, which will require another helper or two in rulesHelpers, which will also need tests. I want to draw the line somewhere on this PR but I don't think we should have effectively new functions without tests.

Reply:
Here are two commits. The first adds tests for `deleteOldEventSourceMappings`. These exist in main on the Rule model. I ported them to the helper and re-created an additional rulesHelper to make these work:
https://github.com/nasa/cumulus/pull/3285/commits/e52bc653bd8cb5bdfe5bee56cb38eb4deaa6bfce

The second commit is a test for the new (old) rules helper:

https://github.com/nasa/cumulus/pull/3285/commits/e52bc653bd8cb5bdfe5bee56cb38eb4deaa6bfce

The helper exists on main but there were no tests so this is new.